i18nilize/build/lib/internationalize/command_line.py

```diff
@@ -0,0 +1,47 @@
+#from src.internationalize.helpers import add_language
+import json
```

Comment:
build/lib/internationalize seems to contain exact copies of code (including outdated ones) in other folders. Was this done intentionally?

Reply:
deleted